frontend/src/console/components/Logos/Logos.spec.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```

Comment:
I know you're just following an already set pattern, but IMHO it'd be better to put test files in a separate directory (instead of putting them together with source code).